src/pfs/pfs.proto

```diff
@@ -578,6 +589,8 @@ service API {
   rpc ListRepo(ListRepoRequest) returns (stream RepoInfo) {}
   // DeleteRepo deletes a repo.
   rpc DeleteRepo(DeleteRepoRequest) returns (google.protobuf.Empty) {}
+  // DeleteRepos deletes more than one repo at once.
+  rpc DeleteRepos(DeleteReposRequest) returns (DeleteReposResponse) {}
```

Comment:
I think the semantics should be "delete all repos that I have permissions to delete"? Not sure where we landed on "who gets to delete repos" but if it's a very privileged user it might be that they will always be able to delete all the repos

Reply:
@brendoncarroll, I believe that that the list of repos returned is exactly the list of repos that are deleted. I’ve added a test for this, but am I missing something?

@acohen4, are you saying that an auth failure when deleting a repo should not be an error? I think that makes sense, but I can also imagine that it should be all-or-nothing. This might be a non-technical question though?

Reply:
It looks like another caller could delete a repo in between listing them and performing the deletion.
- Either the internal deleteRepo is idempotent and returns nil for already deleted repos, causing us to return repos that weren't deleted by the call, but which existed at the start of the call.
- Or the internal deleteRepo errors on non-existent repos, and the function returns an error (perhaps unnecessarily).

The actual behavior is arguably less important than what we promise with the API.  We can modify the behavior later within the bounds of whatever we promise.  If it happens to be all or nothing, but we don't want to guarantee then let's add a comment here explaining that.

I don't really have strong opinions on the semantics, just that we document what they are.  And consumers of the PFS API don't have to look at the PFS implementation to reason about what's happening.

Reply:
@brendoncarroll is this still an issue with the updated API semantics? The request doesn't contain the list of repos now, so there won't be an inconsistency between request.repos and response.repos

Reply:
All or nothing and partial completion are both fine.  It looks like all or nothing is difficult to implement.  I just think we should write down the guarantees here on the interface in a comment, so autocomplete shows the comment with the semantics to developers.

Reply:
I added comments spelling out the semantics.